docs/pages/recipes.md

```diff
@@ -6,48 +6,60 @@ permalink: /recipes
 toc: false
 ---
 
-# Singularity Python Converters
+# Singularity Python Recipes
 
-We will here discuss the Singularity Python converters that will help you
-to convert between recipe files. What kind of things might you want to do?
+We will here discuss the Singularity Python recipe writers and parsers that will
+help you to convert between Singularity and Docker recipes. First, let's
+define what these things are:
+
+ - a *Recipe* is a base class that holds general variables and instructions for a container recipe (e.g., environment, labels, install steps).
+ - a *parser* is a class that knows how to read in a special recipe type (e.g., Dockerfile) and parse it into the Recipe class. 
+ - a *writer* is a class that knows how to use a filled in Recipe to write a special recipe type (e.g., Singularity) with the content.
+
+Now we can answer what kind of things might you want to do:
 
  - convert a Dockerfile to a Singularity Recipe
- - convert a Singularity Recipe to a Dockerfile (TBA)
+ - convert a Singularity Recipe to a Dockerfile
  - read in a recipe of either type, and modify it before doing the above
 
 
 # Command Line Client
+
 You don't need to interact with Python to use the converter! It's sometimes 
 much easier to use the command line and spit something out into the terminal,
 for quick visual inspection or piping into an output file. If you use the 
 `spython` utility, you can see your options available:
 
 
 ```
-spython --help
+spython recipe --help
 
-Singularity Python [v0.0.21]
+usage: spython recipe [-h] [--entrypoint ENTRYPOINT] [--json] [--force]
+                      [--parser {auto,docker,singularity}]
+                      [--writer {auto,docker,singularity}]
```

Comment:
What would auto do?

Reply:
auto is the default, and it detects the recipe type based on the filename. For example, a Dockerfile is detected as needing the "docker" parser, and the "singularity" writer. As long as we have two formats, this is reasonable to do. If there are more than two then the "auto" will be removed from the writer, and we'll force the user to choose.

Reply:
With auto I can do:

```
spython recipe Dockerfile
```
and it will detect docker, and give me Singularity.